arm-web/2016-03-01/DeletedWebApps.json

```diff
@@ -34,92 +34,41 @@
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/topLevelDomains/{name}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/deletedSites": {
       "get": {
         "tags": [
-          "TopLevelDomains"
+          "DeletedWebApps"
         ],
-        "summary": "Gets details of a top level domain",
-        "description": "Gets details of a top level domain",
-        "operationId": "TopLevelDomains_Get",
+        "summary": "Gets deleted web apps in subscription",
+        "description": "Gets deleted web apps in subscription",
+        "operationId": "DeletedWebApps_ListByResourceGroup",
         "consumes": [],
         "produces": [
           "application/json",
-          "text/json",
-          "application/xml",
-          "text/xml"
-        ],
-        "parameters": [
-          {
-            "name": "name",
-            "in": "path",
-            "description": "Name of the top level domain",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "$ref": "#/parameters/subscriptionIdParameter"
-          },
-          {
-            "$ref": "#/parameters/apiVersionParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/TopLevelDomain"
-            }
-          }
-        },
-        "deprecated": false
-      }
-    },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.DomainRegistration/topLevelDomains/{name}/listAgreements": {
-      "post": {
-        "tags": [
-          "TopLevelDomains"
-        ],
-        "summary": "Lists legal agreements that user needs to accept before purchasing domain",
-        "description": "Lists legal agreements that user needs to accept before purchasing domain",
-        "operationId": "TopLevelDomains_ListAgreements",
-        "consumes": [
-          "application/json",
-          "text/json",
-          "application/xml",
-          "text/xml",
-          "application/x-www-form-urlencoded"
-        ],
-        "produces": [
-          "application/json",
           "text/json"
         ],
         "parameters": [
           {
-            "name": "name",
+            "name": "resourceGroupName",
             "in": "path",
-            "description": "Name of the top level domain",
+            "description": "Name of resource group",
             "required": true,
             "type": "string"
           },
           {
-            "name": "agreementOption",
-            "in": "body",
-            "description": "Domain agreement options",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/TopLevelDomainAgreementOption"
-            }
+            "name": "propertiesToInclude",
+            "in": "query",
+            "description": "Additional web app properties included in the response",
```

Comment:
this very generic statement, how an SDK user will know which properties can be included, what will come automatically.